src/PathException.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace Beaver\Dateval;
+
+class PathException extends \Exception {
```

Comment:
__[code_sniffer]__
* Opening brace of a class must be on the line after the definition